### ISSUE TITLE
Wrong scalar type raises incorrect message

### ISSUE THREAD

**Issue by lenaorobei** [webonyx/graphql-php#497]
Wrong scalar type causes `Internal server error` message

Version: `"webonyx/graphql-php": "0.12.6"`

### Steps to reproduce
Use any query/mutation with specific expected type.
```
deleteCustomerById(id: Int!)
```
```
mutation {
  deleteCustomerById(id: "")
}
```
### Expected result
`Expected type Int!, found "".`

### Actual result (*)
`Internal server error`.

### Details
Error message is correct in the runtime, but `\GraphQL\Type\Definition\EnumType::parseLiteral` throws `\Exception` and `\Exception::__construct` checks `else if ($previous)`  and changes the error category to `CATEGORY_INTERNAL`. Then `\GraphQL\Executor\ExecutionResult::toArray` transforms error message to `Internal server error`.

However, validation works fine with complex types.

**Comment by vladar** [comment]
I believe this has been fixed in 0.13.0. Have you tried the latest version?

**Comment by vladar** [comment]
Feel free to re-open with new details if it is still happening for you on 0.13.x
